release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -466,19 +466,115 @@ With this update, the default credentials request for AWS has been modified to a
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-top-aware_persistent-storage-csi-vsphere[vSphere CSI topology].
+
+[id="ocp-4-13-storage-multiple-default-sc"]
+==== Creating more than one default storage class is generally available
+{product-title} 4.13 allows you create more than one default storage class. This feature makes it easier to change the default storage class because you can create a second storage class defined as the default. You then temporarily have two default storage classes before removing default status from the previous default storage class. While it is acceptable to have multiple default storage classes for a short time, you should ensure that eventually only one default storage class exists.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#change-default-storage-class_persistent-storage-csi-sc-manage[Changing the default storage class] and xref:../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#multiple-default-storage-classes[Multiple default storage classes].
+
+[id="ocp-4-13-storage-managing-default-sc"]
+==== Managing the default storage class is generally available 
+{product-title} 4.13 introduces the `spec.storageClassState` field in the `ClusterCSIDriver` object, which allows you to manage the default storage class generated by {product-title} to accomplish several different objectives:
+
+* When you have other preferred storage classes, preventing the storage operator from re-creating the initial default storage class.
+
+* Renaming, or otherwise changing, the default storage class
+
+* Enforcing static provisioning by disabling dynamic provisioning.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc[Managing the default storage class].
+
+[id="ocp-4-13-storage-retroactive-default-sc-assignment"]
+==== Retroactive default StorageClass assignment (Technology Preview)
+Previously, if there was no default storage class, persistent volumes claims (PVCs) that were created that requested the default storage class remained stranded in the pending state indefinitely, unless you manually delete and recreate them. {product-title} can now retroactively assign a default storage class to these PVCs, so that they do not remain in the pending state. With this feature enabled, after a default storage class is created, or one of the existing storage classes is declared the default, these previously stranded PVCs are assigned to the default storage class.
+
+This feature is supported with Technology Preview status.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc[Managing the default storage class].
```

Comment:
@lpettyjo Similar to the [change](https://github.com/openshift/openshift-docs/pull/56097/files#r1189186657) above, I'd reference the [actual feature](https://github.com/openshift/openshift-docs/blob/6d34f08dd0d6ed73a12e8d6318b898d4610ed179/modules/persistent-storage-csi-sc-multiple-none.adoc?plain=1#L12) (titled "Absent default storage class") here too. That way users get to relevant feature details directly.

Otherwise SC feature notes lgtm.

Reply:
+1 let's be consistent with the upstream name

Reply:
Done.